rebar.config

```diff
@@ -21,14 +21,23 @@
 {profiles, [
     {dev, [
         {plugins, [
-            {gpb, "4.3.1"},
-            {rebar3_gpb_plugin, "2.12.0"}
+            {rebar3_gpb_plugin, "2.20.1"}
         ]},
         {provider_hooks, [
             {pre, [
                 {compile, {protobuf, compile}},
                 {clean, {protobuf, clean}}
             ]}
+        ]},
+        {dialyzer, [
+            {warnings, [
```

Comment:
If we add `underspecs` it shows more stuff. Let me know if it would be better to have it.